project/BlenderWang9487/README.rst

```diff
@@ -0,0 +1,106 @@
+========================
+DummyML
+========================
+
+
+
+Basic Information
+=================
+
+A dummy ML library for NSD course term project.
+
+Here is my Github repository:
+`DummyML <https://github.com/BlenderWang9487/DummyML.git>`_
+
+Problem to Solve
+================
+
+I want to build a machine learning library based on some mathematical theories.
+
+There are many easy-to-use hybrid frameworks that built for deep learning, 
+but fewer for other traditional ML algorithms. So I'm going to build one.
+
+Some regression, classification and clustering models (like Gaussian linear 
+regression, naive Bayes classifier, Kmeans, etc.) will be provided by this
+library. (There will be atleast three approaches for each mission. One of them
+is a "dummy" approach which can be treated as the baseline.)
+
+Prospective Users
+=================
+
+People who are interested in machine learning and want to try some different 
+ML algorithms may use my library.
+
+Data scientists can use this library's models as baselines against theirs, too.
+
+System Architecture
+===================
+
+There may or may not be a class to handle input/output data (I might just use
+Numpy array).
+
+There will be a class, "Model", which is the parent class of all the other
+models. It will provide an uniform interface for fitting, predicting, saving and
+loading.
+
+If there are some functions/data structures which are used by many algorithms
+(like matrix operations), they will be declared in utils.hpp, making them easy
+to maintain.
+
+API Description
+===============
+
+Users can write something like this:
+
+.. code:: python
+    :number-lines:
+
+    import dummyml
```

Comment:
How about using a fancy module name?  There's nothing wrong with `dummyml`, but of course you can name it whatever you want. :)

Reply:
For now, I think it looks good to me. 🥲

But I will think about it. Thanks!